Features/bootstrap/MainContext.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+use QATools\BehatExtension\Context\QAToolsContext;
+
+class MainContext extends QAToolsContext
+{
+
+	/**
+	 * @Given I'm on ':page'
+	 */
+	public function openPage($page)
+	{
+		$this->page = $this->pageFactory->getPage($page);
+		$this->page->open();
+	}
+
+	/**
+	 * @When I login with ':username' and ':password'
+	 */
+	public function login($username, $password)
+	{
+		$this->page->login($username, $password);
+	}
+
+	/**
+	 * @Then I should be on ':page'
+	 */
+	public function pageOpen($page)
+	{
+		$this->page = $this->pageFactory->getPage($page);
```

Comment:
Is there any specific reason to assign `$this->page` also from here (the `@Then` part)? I thought that only assignment should happen from `@Given` part.